store/storetest/post_store.go

```diff
@@ -471,12 +471,34 @@ func testPostStoreGetSingle(t *testing.T, ss store.Store) {
 	o2, err = ss.Post().Save(o2)
 	require.NoError(t, err)
 
+	o3 := &model.Post{}
+	o3.ChannelId = o1.ChannelId
+	o3.UserId = o1.UserId
+	o3.Message = "zz" + model.NewId() + "d"
+	o3.RootId = o1.Id
+
+	o4 := &model.Post{}
+	o4.ChannelId = o1.ChannelId
+	o4.UserId = o1.UserId
+	o4.Message = "zz" + model.NewId() + "e"
```

Comment:
We should use the `model.NewRandomString` method here and above (line 477)

Reply:
It's better to use the `NewTestId` function for this. That was specifically made to remove this "zz" hack.

Reply:
Here we just need a random string for the post message body, so I thought `NewRandomString` would be good enough.

Reply:
Ahh sure, that's fine. :+1:

Reply:
I just followed the pattern of other objects o1 and o2  for message body, but sure, I'll change it to NewRandomString()